src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -357,21 +357,48 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
 
   render() {
     const { artwork, inquiryComponent } = this.props
+    const isPriceListed = !artwork?.isPriceHidden
+    const isOfferableFromInquiry = artwork?.isOfferableFromInquiry
+    const isOfferable = artwork?.is_offerable
+    const isAcquireable = artwork?.is_acquireable
+    const isInquireable = artwork?.is_inquireable
 
     const {
       isCommittingCreateOrderMutation,
       isCommittingCreateOfferOrderMutation,
       selectedEditionSet,
     } = this.state
 
-    const artworkEcommerceAvailable = !!(
-      artwork.is_acquireable || artwork.is_offerable
-    )
+    const artworkEcommerceAvailable = !!(isAcquireable || isOfferable)
 
-    if (!artwork.sale_message && !artwork.is_inquireable) {
+    if (!artwork.sale_message && !isInquireable) {
       return <Separator />
     }
 
+    const shouldDisplayMakeOfferButton = (): boolean => {
+      if (isOfferable || (isPriceListed && isOfferableFromInquiry)) {
```

Comment:
I don't think we have to be checking for `isPriceListed` here, no? 🤔 
Also, I think here should be the lab flag check; something like `if (isOfferable || (isOfferableFromInquiry && hasFeatureFlag("Make Offer On All Eligible Artworks"))) {` (or however we check for those 😅 ).

Reply:
Also, we can probably simplify this to directly return that condition.

Reply:
> I don't think we have to be checking for isPriceListed here, no? 🤔

Looks like we could use either field, checking if listPrice.price is null or for the value of this boolean. I think it's a case of 2 +2 = 4 as is 3 + 1 here unless there's some reason why one check over the other is valid?
https://ibb.co/XpxYT90

Reply:
I asked about this in Slack [here](https://artsy.slack.com/archives/C02JHHHKP5K/p1640269454255200?thread_ts=1640206269.250700&cid=C02JHHHKP5K) and now I think checking if the price is listed might be the way to go?

Reply:
Hmm I'm not sure if we care. At least, I think, on Inquiry Checkout we're not checking if the artwork has a listed price, only for `isOfferableFromInquiry`, and the logic should be the same, no? Maybe I'm missing something 🤔

Reply:
I was going off of the acceptance criteria in the [ticket](https://artsyproduct.atlassian.net/browse/NX-3042):

`MO button shows on artwork page of artworks currently available through MOI as long as price display is exact price or price range.`